### PR TITLE
Resolve of #136 & #129

### PR DESCRIPTION
**Pull request by Denio21**

### CODE REVIEW

File: languages/en.json [hunk 1]
```diff
@@ -284,10 +288,13 @@
     "builder.event.create.role.help": "Enter name of role for event is.",
     "builder.event.create.place.title": "Place of event",
     "builder.event.create.place.help": "Enter place of the event.",
+    "builder.event.create.place.example": ["School", "Wall Street", "Wenceslas Square"],
```
Comment (cajthamlmatej) [inline languages/en.json]: You are not allowed to do this. We are not accepting arrays of string in translations. Do https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/split (of comma ",") and then https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/join of string: so "School,Wall Street,Wenceslas Square". Do it in all of arrays that you added to translations.

File: src/commands/EventCommand.js [hunk 2]
```diff
@@ -419,15 +425,15 @@ class EventCommand extends SubsCommand {
             const events = startsEvents.concat(endsEvents).concat(goingEvents);
 
             let string = "";
-            if(events != null)
+            if (events != null)
                 events.forEach(event => {
                     string += "**" + event.name + "** - " + channel.guild.roles.find(r => r.id == this.eventModule.getMentionableRolesIds()[event.role]) + "\n";
```
Comment (cajthamlmatej) [inline src/commands/EventCommand.js]: If we already have title, we can change all references to "name". Name should be just used for editing purposes.

Comment (cajthamlmatej) [reply src/commands/EventCommand.js]: Please search for others references to name! I think we use it even in event annoucement.

File: src/commands/EventCommand.js [hunk 3]
```diff
@@ -70,66 +70,73 @@ class EventCommand extends SubsCommand {
...
                 "validate": (content) => {
-                    if (!types.includes(content)) 
+                    if (!types.includes(content))
                         return ["command.event.type-not-valid", types.join(", ")];
                     else
                         return true;
                 }
             },
             {
                 "name": "name",
```
Comment (cajthamlmatej) [inline src/commands/EventCommand.js]: Can we please movetitle before name? Can we please add option "-" to name to auto-generate the name from the title (replacing spaces to -, making all chars lowercase and replacing non-basic chars (ř -> r))? Example of using aut-generated options, please look into this same file and lin 126 and lower.

File: src/commands/EventCommand.js (hunk 2 above)
Comment (Denio21) [reply src/commands/EventCommand.js]: Can we make this one as a separate issue?

File: src/commands/EventCommand.js [hunk 4]
```diff
@@ -70,126 +70,143 @@ class EventCommand extends SubsCommand {
...
+            "value": (content, values) => {
+                if (content == "-")
+                    return values["title"].toLowerCase().split("").map(function(letter) {
+                        const i = this.accents.indexOf(letter);
+                        return (i !== -1) ? this.acc_out[i] : letter;
+                    }.bind({
+                        accents: "àáâãäåąßòóôőõöøďdžěèéêëęðçčćìíîïùűúûüůľĺłňñńŕřšśťÿýžżźž- ",
+                        acceOut: "aaaaaaasoooooooddzeeeeeeeccciiiiuuuuuulllnnnrrsstyyzzzz__"
+                    })).join("");
+            }
```
Comment (cajthamlmatej) [inline src/commands/EventCommand.js]: This method changes the value. You need to return content; if content is  not -.
```suggestion
                else
                    return content;
            }
```

**Pull request merged**